wetmap/src/components/newModals/tripCreatorListModal/view.tsx

```diff
@@ -0,0 +1,60 @@
+import React, { useRef } from 'react';
+import Itinerary from '../../itineraries/itinerary';
+import WavyModalHeader from '../../reusables/wavyModalHeader';
+import Button from '../../reusables/button';
+import style from './style.module.scss';
+import defaultHeaderPicture from '../../../images/blackManta.png';
+import { ItineraryItem } from './types';
+import { DiveShop } from '../../../entities/diveShop';
+
+
+type TripCreatorListViewProps = {
+  setSelectedID:                (id: number) => void
+  handleBackButton:             () => void
+
+  diveShop:         DiveShop | null
```

Comment:
Why are you bringing this prop in? doesn't seem needed?

Reply:
Thanks, was just an artifact of laziness and was going to remove after further implementation

Reply:
removed